metaworld/policies/mujoco/sawyer_xyz/sawyer_drawer_open.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
```

Comment:
packages should always be named by their main class

`sawyer_drawer_open_v1_policy.py`